src/annotator/util/test/shadow-root-test.js

```diff
@@ -29,5 +29,21 @@ describe('annotator/util/shadow-root', () => {
       assert.ok(styleEl);
       assert.match(styleEl.textContent, /@import ".*annotator\.css.*"/);
     });
+
+    it('does not injects stylesheets into the shadow root if style is not found', () => {
+      const container = document.createElement('div');
+
+      const link = document.querySelector(
+        'link[rel="stylesheet"][href*="/build/styles/annotator.css"]'
+      );
+      // Removing the `rel` attribute is enough for the URL to not be found
+      link.removeAttribute('rel');
+
+      createShadowRoot(container);
+
+      const styleEl = container.shadowRoot.querySelector('style');
+      assert.isNull(styleEl);
+      link.setAttribute('rel', 'stylesheet');
```

Comment:
This could potentially cause unexpected failures in other tests if this test fails. In this case I think we're very likely to be safe since other tests generally depend on the CSS rather than the element and the styles will have already loaded by this point.